experiment/build/generate_cloudbuild.py

```diff
@@ -13,64 +13,115 @@
 # limitations under the License.
 """Generates Cloud Build specification"""
 
-import argparse
-import json
-import yaml
-
-from experiment.build import docker_images
-
-
-# TODO(Tanq16): Add unit test for this.
-def create_cloud_build_spec(buildable_images, docker_registry):
-    """Returns Cloud Build specificatiion."""
-
-    cloud_build_spec = {}
-    cloud_build_spec['steps'] = []
-    cloud_build_spec['images'] = []
-
-    for name, image in buildable_images.items():
-        step = {}
-        step['id'] = name
-        step['name'] = 'gcr.io/cloud-builders/docker'
-        step['args'] = []
-        step['args'] += ['--tag', image['tag']]
-        step['args'] += ['--cache-from', docker_registry + image['tag']]
-        step['args'] += ['--build-arg', 'BUILDKIT_INLINE_CACHE=1']
-        if 'build_arg' in image:
-            for build_arg in image['build_arg']:
-                step['args'] += ['--build-arg', build_arg]
-        if 'dockerfile' in image:
-            step['args'] += ['--file', image['dockerfile']]
-        step['args'] += [image['context']]
-        if 'depends_on' in image:
-            step['wait_for'] = []
-            for dep in image['depends_on']:
-                step['wait_for'] += [dep]
-        cloud_build_spec['images'].append(name)
+import os
+import posixpath
+
+from common import yaml_utils
+from common import experiment_utils
+from common import experiment_path as exp_path
+from common.utils import ROOT_DIR
+from experiment.build import build_utils
+
+BASE_TAG = 'gcr.io/fuzzbench'
+
+
+def get_experiment_tag_for_image(image_specs, is_tagged=True):
```

Comment:
maybe tag_by_experiment instead of is_tagged.

Reply:
Done